cats-frontend/src/api/mice.api.ts

```diff
@@ -0,0 +1,10 @@
+export const deleteMouseApi = async (id: number): Promise<void> => {
+    const response = await fetch(`http://localhost:3001/mice/${id}`, {
```

Comment:
```suggestion
    const response = await fetch(`${BASE_URL}/mice/${id}`, {
```

Add a `BASE_URL` variable and use it